apps/desktop/desktop/src/main/java/bisq/desktop/main/content/bisq_easy/open_trades/trade_state/states/SellerStateLightning3b.java

```diff
@@ -103,12 +102,17 @@ public void add(BisqEasyOpenTradeMessage message) {
                                 String expectedEncoded = Res.encode("bisqEasy.tradeState.info.buyer.phase3b.tradeLogMessage.ln", peersUserName);
                                 boolean shouldUpdateUI = encodedLogMessage.equals(expectedEncoded);
                                 if (!shouldUpdateUI) {
-                                    String clearedPeersUserName = UserProfileService.getClearedUserName(peersUserName);
-                                    if (!clearedPeersUserName.equals(peersUserName)) {
-                                        String simplifiedExpectedEncoded = Res.encode("bisqEasy.tradeState.info.buyer.phase3b.tradeLogMessage.ln",
-                                                clearedPeersUserName);
-                                        shouldUpdateUI = encodedLogMessage.equals(simplifiedExpectedEncoded);
-                                    }
+                                    /*
```

Comment:
The comment is not describing the context adequately. "multiple users share that nickname" - it is just that user can choose nicknames freely thus multiple could come up with the same nickname. "sharing" is misleading.